server/lib/nl/config_builder/base.py

```diff
@@ -75,6 +75,53 @@ def _replace_special(input_string_title_case: str) -> str:
   return input
 
 
+def _count_caps(word: str) -> int:
+  return sum(1 for c in word if c.isupper())
+
+
+def _make_sentence_case(input_string: str) -> str:
+  if not input_string:
+    return ""
+
+  output_str = ""
+  sentences = input_string.split(".")
```

Comment:
is having a stale `.`, like RCP 4.5 OK?

Reply:
yes it is :) but this is currently removed and will add in the next PR